scripts/trace_flowcell.py

```diff
@@ -2,117 +2,145 @@
-    for trace in traces:
-        trace_id = trace["id"]
-        trace_info_response = requests.get(
-            "{}/{}/{}".format(api_base_url, "traces", trace_id),
-            verify=False,
-            headers=headers)
-        trace_info = json.loads(trace_info_response.text)
-        action_executions = map(lambda x: x["object_id"], trace_info["action_executions"])
-        for execution in action_executions:
-            yield execution
-
-def get_actions_from_executions(executions, headers):
-    for execution_id in executions:
-        execution_info_response = requests.get(
-            "{}/{}/{}".format(api_base_url, "executions", execution_id),
-            verify=False,
-            headers=headers)
-        execution_info = json.loads(execution_info_response.text)
-        yield execution_info
-
-def filter_actions_by_name(actions, name):
-    return (action for action in actions if name in action["action"]["name"])
-
-def sort_actions_by_timestamp(actions):
-    def get_start_time(action):
-        #Select start_time, e.g. 2017-05-29T15:03:03.818222Z for each action. Remove everything after ".".
-        start_time = datetime.strptime(action['start_timestamp'].split(".")[0], "%Y-%m-%dT%H:%M:%S")
-        return start_time
-    #Compare elements in actions on key value start_timestamp.
-    sort_actions = sorted(actions, key=get_start_time)
-    return sort_actions
 
 if __name__ == "__main__":
 
-    parser = argparse.ArgumentParser(description="Gets execution ids associated with a trace tag (e.g. a runfolder) and a "
-                                             "workflow. It can be used to track executions, e.g: "
-                                             "python scripts/trace_runfolder.py --tag 150605_M00485_0183_000000000-ABGT6_testbio14 | xargs -n1 st2 execution get")
-    parser.add_argument('--flowcell', required=True)
-    parser.add_argument('--api_base_url', default="https://localhost:4041/api/v1")
-    parser.add_argument('--workflow', default="workflow")
+    # Try to load access token from environment - fall back if not available
+    try:
+        access_token = os.environ['ST2_AUTH_TOKEN']
+    except KeyError as e:
+            print("Could not find ST2_AUTH_TOKEN in environment. "
+                "Please set it to your st2 authentication token.")
+    access_headers = {"X-Auth-Token": access_token}
+        
+    parser = argparse.ArgumentParser(description="Gets execution ids associated with a flowcell "
+                                     "(e.g. a runfolder) and a workflow. It can be used to track "
+                                     "executions, e.g: python scripts/trace_flowcell.py "
+                                     "--flowcell 000000000-ABGT6_testbio14 ")
+    parser.add_argument('--flowcell',     required=True)
+    parser.add_argument('--noverify',     action="store_false")
+    parser.add_argument('--api_base_url', default="http://localhost:9101/v1")
```

Comment:
I think it's a typo here and the default should be `http://localhost:9101/api/v1`